src/middlewared/middlewared/plugins/ldap.py

```diff
@@ -457,8 +454,8 @@ async def common_validate(self, new, old, verrors):
                 "Bind credentials or kerberos keytab are required for an authenticated bind."
             )
         if new["bindpw"] and new["kerberos_principal"]:
-            self.logger.warning("Simultaneous keytab and password authentication "
-                                "are selected. Clearing LDAP bind password.")
+            self.logger.info("Simultaneous keytab and password authentication "
+                             "are selected. Please clear LDAP bind password.")
```

Comment:
If this is called when validating ldap.update() then we in theory will now update with the bindpw removed. I think it's probably fine to just remove logging here.

Reply:
Agreed